content/docs/nav.yml

```diff
@@ -95,84 +94,84 @@
     - id: react-dom-server
       title: ReactDOMServer
     - id: dom-elements
-      title: DOM Elements
+      title: Elemente de DOM
     - id: events
       title: SyntheticEvent
     - id: test-utils
-      title: Test Utilities
+      title: TestUtilities
     - id: shallow-renderer
-      title: Shallow Renderer
+      title: ShallowRenderer
     - id: test-renderer
-      title: Test Renderer
+      title: TestRenderer
     - id: javascript-environment-requirements
-      title: JS Environment Requirements
+      title: Cerințe pentru mediul JS
     - id: glossary
       title: Glossary
 - title: Hooks
   isOrdered: true
   items:
     - id: hooks-intro
-      title: Introducing Hooks
+      title: Introducere Hooks
     - id: hooks-overview
-      title: Hooks at a Glance
+      title: Hooks pe scurt
     - id: hooks-state
-      title: Using the State Hook
+      title: Folosirea Hook-ului State
     - id: hooks-effect
-      title: Using the Effect Hook
+      title: Folosirea Hook-ului Effect
     - id: hooks-rules
-      title: Rules of Hooks
+      title: Reguli ale Hook-urilor
     - id: hooks-custom
-      title: Building Your Own Hooks
+      title: Construieste-ti propriul Hook
```

Comment:
Aici am schimbat eu din Carlige in Hooks ca a fost conflict cu master-ul.. ma gandeam ca e putin mai ok. Ce parere aveti @gabrielmoncea @nikiciv ?